docs/sdks/client-sdks/javascript/Initialization.mdx

```diff
@@ -79,6 +79,15 @@ Poll for new **configurations** (every 30 seconds) after successfully requesting
 Poll for new configurations even if the initial configurations request failed.
 </ApiOptionRef>
 
+<ApiOptionRef
+  name="pollingIntervalMs"
```

Comment:
Proof that we provide the ability to pass this in when initializing the JS client

https://github.com/Eppo-exp/js-client-sdk/blob/6fc8683509c44a8cd168b81f380e98a4444f4907/src/index.ts#L381